### ISSUE TITLE
hey is this good for beginners?

### ISSUE THREAD

**Issue by deudz** [TodePond/GulfOfMexico#431]
would this language be easier to learn than python?

**Comment by TodePond** [comment]
yes

**Comment by Justiniscoding** [comment]
You know this is a joke right? Learn Javascript or python for actual use cases

**Comment by mybearworld** [comment]
It doesn't force people to memorize huge amounts of keywords - for example, you can just declare a function like this:
```java
n add x => x + x!
```

**Comment by Justiniscoding** [comment]
Yeah but the only way to run it is in chatgpt. I find this really funny, it's just that if you are actually considering learning to program, you should learn javascript or python. If you want to fool around, definitely learn this or another eccentric programming language. Another fun one to learn is LOLCODE, which has very *interesting* keywords/reserved words
